Google.Api.Generator.Tests/ProtoTest.cs

```diff
@@ -105,8 +105,16 @@ public void TestThatSomeCodeIsGenerated()
             Assert.NotNull(ns);
             // Check there is one settings class.
             Assert.NotEmpty(ns.Members);
-            var settingsCls = ns.Members.Where(x => (x as ClassDeclarationSyntax)?.Identifier.Text == "TestSettings").ToList();
-            Assert.Single(settingsCls);
+            var settingsClss = ns.Members.OfType<ClassDeclarationSyntax>().Where(x => x.Identifier.Text == "TestSettings").ToList();
```

Comment:
Any reason not to use `settingsClasses` and `settingsClass` here? I feel this is brevity at the expense of readability.